operations/observability/mixins/workspace/rules/components/nodes/alerts.libsonnet

```
@@ -48,6 +48,21 @@
               increase(cluster_autoscaler_failed_scale_ups_total[1m]) != 0
             |||,
           },
+          {
+            alert: 'NetworkConnectionsTooHigh',
+            labels: {
+              severity: 'critical',
+            },
```

```suggestion
labels: {
              severity: 'warning',
              team: 'workspace'
            },
```

This way it routes to https://gitpod.slack.com/archives/C03QZ1NH517, and we can gauge if this is too frequent.

---

:point_up: because I don't want to send the alert to on-callers until we (on the workspace team) have felt it first in Slack.